stories/components/Radio/radioGroupMinimal.md

```diff
@@ -98,8 +98,10 @@ An icon can be provided. If a label is not provided, but an aria-label is, that
 will be used as the tooltip message.
 
 ```jsx
-<RadioGroupMinimal aria-label="Option 1" icon={User} label="" />
-<RadioGroupMinimal icon={User} label="Option 1" />
+<RadioGroupMinimal>
```

Comment:
Fixed docs here. I think we want `<Radio` vs `<RadioGroupMinimal`

Reply:
Ahh good catch!